app/src/main/java/com/codingblocks/cbonlineapp/fragments/QuizFragment.kt

```diff
@@ -122,6 +131,48 @@ class QuizFragment : Fragment(), AnkoLogger, ViewPager.OnPageChangeListener, Vie
             count++
             rowLayout.addView(numberBtn)
         }
+        val submitButton = Button(context)
+        submitButton.background = context!!.getDrawable(R.drawable.submit_button_background)
+        submitButton.textColor = context!!.resources.getColor(R.color.white)
+        val params = LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT, LinearLayout.LayoutParams.WRAP_CONTENT)
+        params.bottomMargin = 16
+        submitButton.setCompoundDrawablesWithIntrinsicBounds(0, 0, R.drawable.submit_image, 0)
+        submitButton.compoundDrawablePadding = 16
+        params.topMargin = 8
+        // submitButton.backgroundColor = context!!.resources.getColor(R.color.colorPrimaryDark)
+        submitButton.layoutParams = params
+        submitButton.text = "Submit"
+        submitButton.setOnClickListener {
+            confirmSubmitQuiz()
+        }
+        numberLayout.addView(submitButton)
+    }
+
+    private fun confirmSubmitQuiz() {
+        val builder = AlertDialog.Builder(activity)
+        val inflater = requireActivity().layoutInflater
+        val customView = inflater.inflate(R.layout.custom_confirm_dialog, null)
+        customView.okBtn.text = "Yes"
+        customView.cancelBtn.text = "Cancel"
+        customView.description.text = "Are you sure to submit the quiz?"
+        builder.setCancelable(false)
+        builder.setView(customView)
+        val dialog = builder.create()
+        customView.cancelBtn.setOnClickListener {
+            dialog.dismiss()
+        }
+        customView.okBtn.setOnClickListener {
+            Clients.onlineV2JsonApi.sumbitQuizById(quizAttemptId).enqueue(retrofitCallback { throwable, response ->
+                val fragmentManager = fragmentManager!!
```

Comment:
make a util function for this and use that in the future to handle fragment transactions(this is optional you may do it if you feel like)

Reply:
done